index.js

```diff
@@ -136,18 +130,12 @@ function start() {
       let gPoint = guestPoint.textContent;
       clearInterval(interval);
       eachRoundScoreDisplay(rounds, hPoint, gPoint);
-         
-      if (rounds < 4){
-        showMessage(message, "Round " +rounds+ " Over");
-      }
-      else {
-        showMessage(message, "Match Over");
-      }
+
+      const roundMsg = rounds < 4 ? showMessage(message, "Round " +rounds+ " Over") : showMessage(message, "Match Over");
```

Comment:
Here the showMessage function is being called in both scenarios so you can create a variable and assign the message in each condition. and outside the condition checks, you can showMessage and pass the variable to it

Reply:
Changed